plugins/transfer/transfer.plugin.zsh

```diff
@@ -0,0 +1,20 @@
+function transfer() {
+  if [ $# -eq 0 ]; then
+    echo "No arguments specified. Usage:\necho transfer /tmp/test.md\ncat /tmp/test.md | transfer test.md"
```

Comment:
You can make this a multi-line "here doc" using `<<<EOF` instead of embedded `\n` for better readability. And should that extra `echo` be inside the string?

```
$ transfer
No arguments specified. Usage:
echo transfer /tmp/test.md
cat /tmp/test.md | transfer test.md
```